### PR TITLE
Consistent naming for pileup functions

### PR DESCRIPTION
**Pull request by jayhesselberth**
Rename `create_se()` to `merge_pileups()` to be consistent with `get_pileup()`

Add family tag for pileup functions

### CODE REVIEW

File: R/pileup.R [hunk 1]
```diff
@@ -56,8 +56,10 @@
 #' @importFrom GenomicRanges GRanges
 #' @importFrom IRanges IRanges
 #' @importFrom GenomeInfoDb seqlevels seqinfo seqlengths
-#' @importFrom BiocParallel SerialParam bpstop bplapply
-#' @rdname get_pileup
+#' @importFrom BiocParallel SerialParam bpstop bplapply ipcid ipclock ipcunlock
```
Comment (kriemo) [inline R/pileup.R:59]: the ipcid, ipclock, and ipcunlock functions aren't used. these can be removed.

File: R/pileup.R [hunk 2]
```diff
@@ -56,8 +56,10 @@
...
+#'
+#' @family pileup
+#'
 #' @export
 get_pileup <- function(bamfiles,
```
Comment (kriemo) [inline R/pileup.R:64]: We should consider renaming this function to something more descriptive like `pileup_sites`. Not necessary in this PR, but something to consider. We however can't use `pileup` as that would conflict with rsamtools.

**Review by kriemo — CHANGES_REQUESTED** [review]

File: R/pileup.R (hunk 2 above)
Comment (jayhesselberth) [reply R/pileup.R:64]: I'm fine with `pileup_sites`. Are the outputs similar between `pileup_sites()` and `pileup_cells()`?

Comment (kriemo) [reply R/pileup.R:64]: the outputs will be similar classes once I've merged #70. pileup_cells returns a SingleCellExperiment and pileup_sites will return a RangedSummarizedExperiment.

Comment (kriemo) [reply R/pileup.R:64]: if you want to merge this in, i'll incorporate the changes, and rename get_pileup in #70.

**Review by kriemo — APPROVED** [review]

**Pull request merged**